src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -3227,15 +3230,15 @@ public void Inheritance_08()
 }";
             var comp = CreateCompilation(source);
             comp.VerifyDiagnostics(
-                // (11,8): error CS0534: 'C' does not implement inherited abstract member 'B.Y.get'
+                // (11,14): error CS0546: 'C.X.init': cannot override because 'A.X' does not have an overridable set accessor
```

Comment:
>// (11,14): error CS0546: 'C.X.init': cannot override because 'A.X' does not have an overridable set accessor [](start = 16, length = 109)

It doesn't look like this error follows specification from the #44618.

It says:
> Each "matching" inherited abstract property's get accessor is overridden.

However it doesn't say anything about init/set accessors. From reading the specification I am getting an impression that we shouldn't be reporting an error for this scenario, we override the getter and leave the init accessor not overriding anything. This, however, isn't something that you can express in a regular C# today. So, it would make sense to not support this for synthesized properties as well. Please, include amended specification in the description of this PR, so that reviewers could review against it.